Content.Server/GameObjects/EntitySystems/VerbSystem.cs

```diff
@@ -85,6 +86,7 @@ private void RequestVerbs(RequestVerbsMessage req, EntitySessionEventArgs eventA
 
             if (!_entityManager.TryGetEntity(req.EntityUid, out var entity))
             {
+                Logger.Warning($"{nameof(RequestVerbs)} called on a client-side entity with id {req.EntityUid} by player {player}.");
```

Comment:
The entity not existing doesn't *necessarily* mean that the entity is client side. 
```suggestion
                Logger.Warning($"{nameof(RequestVerbs)} called on a nonexistant entity with id {req.EntityUid} by player {player}.");
```